openqabot/loader/smelt.py

```diff
@@ -25,7 +25,8 @@
 INCIDENT = '{incidents(incidentId: %(incident)s) { edges { node {emu project \
 repositories { edges { node { name } } } requestSet(kind: "RR") { edges { node \
 { requestId status { name } reviewSet { edges { node { assignedByGroup { name } \
-status { name } } } } } } } packages { edges { node { name } } } } } } }'
+status { name } } } } } } } packages { edges { node { name } } } } } \
+    edges{ node { crd } } } }'
```

Comment:
please update also `INCIDENT_SCHEMA`

Reply:
can you help me with that ?

Reply:
so I added something , not sure if it , right